app/src/main/java/com/esri/arcgismaps/kotlin/sampleviewer/model/SampleCategory.kt

```diff
@@ -26,18 +26,24 @@ enum class SampleCategory(val text: String) {
     VISUALIZATION("Visualization"),
     FAVORITES("Favorites");
 
-    // Return title of each sample in String type
+    /**
+     * Return string of category enum.
+     */
     override fun toString(): String {
         return text
     }
 
+    /**
+     * Return the enum of the category string.
+     */
     companion object {
         fun toEnum(categoryString: String): SampleCategory {
             return entries.firstOrNull {it.text == categoryString}!!
         }
     }
 }
 
+// TODO -- I'm not sure I fully understand why we need to serialize the enum
```

Comment:
Seems like this is used to convert a Kotlin enum type to a simple String, and vice versa 

Probably we could replace this with a `@TypeConverters` as used in the `Converters.kt` file.

Reply:
Oops yes, meant to delete this line